CHANGELOG.md

```diff
@@ -216,7 +216,10 @@ To configure a band scale, use the `Scale` plugin as follows:
 * **react-grid:** extract sorting control ([#881](https://github.com/DevExpress/devextreme-reactive/issues/881)) ([2a6dd1a](https://github.com/DevExpress/devextreme-reactive/commit/2a6dd1a))
 * **react-grid:** support columns fixing ([#1305](https://github.com/DevExpress/devextreme-reactive/issues/1305)) ([81c13a4](https://github.com/DevExpress/devextreme-reactive/commit/81c13a4))
 
+### BREAKING CHANGES
 
+* **react-grid:**
+Because we've extracted the sorting control, the `getMessage` property, which returns the sort label's text, has been moved from the `TableHeaderRow.CellProps` interface to `TableHeaderRow.SortLabelProps`.
```

Comment:
This sentence is awkward because you use too many commas and create multiple subordinate clauses. Rewrite.